conformance/Makefile.am

```diff
@@ -2,7 +2,12 @@
 
 conformance_protoc_inputs =                                    \
   conformance.proto                                            \
-  $(top_srcdir)/src/google/protobuf/test_messages_proto3.proto
+  $(top_srcdir)/src/google/protobuf/test_messages_proto3.proto 
+  
+# proto2 input files, should be separated with proto3, as we 
+# can't generate proto2 files for ruby, php and objc   
```

Comment:
The objc part isn't true.  We support proto2.  I can back fill the conformance support after this lands.